src/components/Transaction/Transaction.re

```diff
@@ -0,0 +1,82 @@
+open Types;
+
+[@react.component]
+let make = (~transaction: state) => {
+  let convert = num => {
```

Comment:
It looks like you are trying to get the absolute value. If yes we can use `abs` for this